src/StatsBase.jl

```diff
@@ -156,7 +156,8 @@ module StatsBase
     residuals,
     r2,
     r²,
-    model_response
+    model_response,
+    ConvergenceException
```

Comment:
Either you should follow the lexical ordering, or you should create a new group. I don't have strong feelings about which choice is best.